README.md

```diff
@@ -730,76 +730,130 @@ classDiagram
     CSSListItem <|-- AtRule: inheritance
     AtRule <|.. AtRuleBlockList: realization
     CSSBlockList <|-- AtRuleBlockList: inheritance
+    AtRuleBlockList ..> OutputFormat: dependency
     AtRule <|.. AtRuleSet: realization
+    AtRuleSet ..> OutputFormat: dependency
```

Comment:
Looks like it's now also picking up dependencies that are method parameters, not just properties.